### PR TITLE
Homepage page layout css

### PR DESCRIPTION
**Pull request by martypowell**
Decided to call it layout, although this may change. It looks like most of this code could be stripped out into component.

### CODE REVIEW

**Comment by martypowell** [comment]
fixes #67

File: stylesheets/partials/layouts/agency-homepage.scss [hunk 1]
```diff
@@ -0,0 +1,138 @@
+/*ADD TO ANIMAL SERVICES FOLDER*/
```
Comment (mxsnyder) [inline stylesheets/partials/layouts/agency-homepage.scss]: General question -- is this just cut and pasted from Sam's stuff? There's no nesting or anything going on here, and I think it's worth a quick refactoring to make it sassier.

Comment (martypowell) [inline stylesheets/partials/layouts/agency-homepage.scss]: @mxsnyder agreed, should be broken into components if possible, as well as removing the red-inside overrides once that is cleaned up. this was just a means of giving sam a quick css file that was source controled until we make it sassier.

**Comment by mxsnyder** [comment]
Verified! You're good to go to merge.

**Pull request merged**